linters/.eslintrc

```diff
@@ -101,7 +101,12 @@
     # FUNCTIONS <https://github.com/thenerdery/javascript-standards#functions>
     ###############################################################################################
 
-    "no-param-reassign": [2, {"props": true}],    # Disallow reassignment of parameters
+    "no-param-reassign": [2, {    				  # Disallow reassignment of parameters
```

Comment:
I think this line (and a few others in this document) is mixing tabs and spaces for indenting the comments.